lib/sgds/sass/_sidenav.scss

```diff
@@ -1,84 +1,95 @@
 .sgds {
-    &.sidenav {
-        width: $sidenav-width;
-        font-size: $sidenav-fontsize;
-        list-style: none;
-        .sidenav-item {
-            +.sidenav-item {
-                margin-top: $sidenav-item-spacing;
-            }
-            .sgds.btn {
-                color: $sidenav-color;
-                font-size: inherit;
+  &.sidenav {
+    --#{$prefix}sidenav-width: #{$sidenav-width};
```

Comment:
small trial in writing css custom variables the 5.3 way. @rav3 any thoughts?

Reply:
Yup this works, possible use cases for the width.
1. use as fix sidenav width in the variable,
or 
2 100% to fit the size of the parent container.